ts/features/payments/home/screens/PaymentsHomeScreen.tsx

```diff
@@ -1,3 +1,50 @@
+import { HeaderFirstLevel, VSpacer } from "@pagopa/io-app-design-system";
+import { useFocusEffect } from "@react-navigation/native";
 import * as React from "react";
+import I18n from "../../../../i18n";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { walletPaymentGetUserWallets } from "../../payment/store/actions/networking";
+import PaymentHistorySection from "../components/PaymentsHomeScreenHistorySection";
+import PaymentMethodsSection from "../components/PaymentsHomeScreenMethodsSection";
+import { isAnySectionSomeOrLoadingSelector } from "../store/selectors";
+import { useTransactionHistory } from "../utils/hooks/useTransactionHistory";
 
-export const PaymentsHomeScreen = () => <></>;
+export const PaymentsHomeScreen = () => {
+  const dispatch = useIODispatch();
+  const { loadFirstHistoryPage } = useTransactionHistory();
+  const shouldRenderEmptyState = !useIOSelector(
+    isAnySectionSomeOrLoadingSelector
+  );
+
+  const fetchData = React.useCallback(() => {
+    dispatch(walletPaymentGetUserWallets.request());
+    loadFirstHistoryPage();
+  }, [dispatch, loadFirstHistoryPage]);
+  useFocusEffect(fetchData);
+
+  if (shouldRenderEmptyState) {
+    return <></>;
+  }
+
+  // let the single components handle empty cases.
+  //
+  // else if neither is some/loading, render empty page.
+
+  return (
+    <>
+      <HeaderFirstLevel
+        title={I18n.t("payment.homeScreen.title")}
+        type="singleAction"
+        firstAction={{
+          accessibilityLabel: I18n.t("payment.homeScreen.title"),
+          icon: "help",
+          onPress: () => null
+        }}
+      />
```

Comment:
Why don't you set the `HeaderFirstLevel` with `react-navigation`? You can easily swap the default one (which I can see in the attached screenshot) and test if everything is ok by temporarily importing the screen into the `TabNavigator`. If you want to give the option to go back (in the developer section), just enable the swipe action.